clients/java/src/main/java/io/camunda/zeebe/client/impl/command/CreateDocumentLinkCommandImpl.java

```diff
@@ -44,12 +47,15 @@ public CreateDocumentLinkCommandImpl(
       final String documentId,
       final String storeId,
       final JsonMapper jsonMapper,
-      final HttpClient httpClient) {
+      final HttpClient httpClient,
+      final ZeebeClientConfiguration configuration) {
+    ensureNotNull("documentId", documentId);
```

Comment:
🔧 I think this check here is not needed, as we instantiate the class in `ZeebeClientImpl`.